Makefile

```diff
@@ -203,6 +213,17 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) container-push IMG=$(BUNDLE_IMG)
 
+.PHONY: operator-sdk
+operator-sdk:
```

Comment:
nice catch, but please let's be consistent to what the Makefile already does for `opm`. So either let's cache all the tools or none of the tools.

Reply:
done